psl/psl-core/src/validate/validation_pipeline/validations/relation_fields.rs

```diff
@@ -151,8 +152,21 @@ pub(super) fn referential_actions(field: RelationFieldWalker<'_>, ctx: &mut Cont
     let msg_foreign_keys = |action: ReferentialAction| {
         let allowed_actions = connector.referential_actions();
 
+        let additional_info = match (action, connector.provider_name()) {
+            (ReferentialAction::SetDefault, "mysql") => Some(formatdoc! {
```

Comment:
No need to change it now, but for future PRs: we try hard to avoid connector-specific code in the core. This should be in the MySQL connector.